Jenkinsfile

```diff
@@ -21,7 +17,18 @@ pipeline {
         }
         stage('Unit Tests') {
             steps {
-                sh 'build/run make -j\$(nproc) check'
+                sh 'build/run make -j\$(nproc) test'
+            }
+        }
+        stage('Integration Tests') {
+            environment {
+                KUBE_VERSION = "v1.6"
+                SUITE = "BlockStorage"
+            }
+            steps {
+                sh 'env | sort'
+                sh 'tests/scripts/kube-dind-cluster.sh up'
```

Comment:
I intentionally leave the cluster behind to speedup bringing it up again. calling `reup` on the dind-cluster is designed to do that. so no need for cleanup.